forge-gui/res/editions/30th Anniversary Edition.txt

```diff
@@ -4,7 +4,7 @@ Date=2022-11-28
 Name=30th Anniversary Edition
 Type=Collector_Edition
 BoosterCovers=3
-Booster=7 Common:fromsheet("30A cards"), 3 Uncommon:fromSheet("30A cards"), 1 Rare:fromSheet("30A cards"), 2 BasicLand:fromSheet("30A cards"), 1 fromSheet("30A retrolands"), 1 fromSheet("30A retrocards")
+Booster=7 Common:fromsheet("30A cards"), 3 Uncommon:fromSheet("30A cards"), 1 Rare:fromSheet("30A cards"), 2 BasicLand:fromSheet("30A cards"), 1 BasicLand:fromSheet("30A retro frame"), 1 !BasicLand:fromSheet("30A retro frame")
```

Comment:
why is the last slot now nonBasicLand? was this an error?

Reply:
Before, there was a section "retrolands" with all the retro frame basic land and a section "retrocards" with all the retro frame non basic land.
I have just simplified using 1 BasicLand:fromSheet("30A retro frame") and 1 !BasicLand:fromSheet("30A retro frame")

Reply:
Now that correct art index is set in CardEdition.java#512, there is no need to additional section that contains the art index. Indicating "1 BasicLand:fromSheet("30A retro frame")" will take the correct art index.

Reply:
ah I understand
I'm not sure if that slot is really limited to nonBasic but this isn't my best area anyway :)

Reply:
There was only basic lands in the former section "30A retrolands", so yes it is only limited to basic lands :)